x-pack/plugins/maps/public/classes/styles/heatmap/heatmap_style.tsx

```diff
@@ -85,6 +85,8 @@ export class HeatmapStyle implements IStyle {
       radius = 64;
     } else if (resolution === GRID_RESOLUTION.MOST_FINE) {
       radius = 32;
+    } else if (resolution === GRID_RESOLUTION.SUPER_FINE) {
```

Comment:
Is this needed? logic in es_geo_grid_source throws an error if grid resolution is super_fine for heatmap layers